docs/sources/setup-grafana/configure-security/planning-iam-strategy.md

```diff
@@ -0,0 +1,193 @@
+
+Organize users in subgroups that are sensible to the organization. These are some examples:
+
+- **Security**: Different groups of users or customers should only have access to their intended resources.
+- **Simplicity**: Reduce the scope of dashboards and resources available.
+- **Cost attribution**: Track and bill costs to individual customers, departments, or divisions.
+- **Customization**: Each group of users could have a personalized experience like different dashboards or theme colours.
+
+### Users in Grafana teams
+
+You can organize users into [teams] and assign them roles and permissions reflecting the current organization. For example, instead of assigning five users access to the same dashboard, you can create a team that consists of those users and assign dashboard permissions to the team.
+
+A user can belong to multiple teams, and can be a member or an administrator for a given team. Members of a team inherit permissions from the team, but they cannot edit the team itself. Team administrators can add members to a team and update its settings, such as the team name, team members, roles assigned to the team, and UI preferences.
+
+Teams are a perfect solution for working with a small subset of users. Teams can share resources with other teams.
+
+### Users in Grafana organizations
+
+[Grafana organizations] allow complete isolation of resources, such as dashboards and data sources. Users can be members of one or several organizations, and they can only access resources from an organization that they belong to.
+
+Having multiple organizations under a single instance of Grafana allows you to manage your users in one place, while having a complete separation of resources.
+
+Organizations provide a higher measure of isolation within Grafana than teams do, and can be useful in certain scenarios. However, because organizations lack the scalability and flexibility of teams and [folders], we do not recommend using them as the default way to group users and resources.
+
+Note that Grafana Cloud does not support having several organizations per instance.
+
+### Choosing between teams and organizations
+
+[Grafana teams] and Grafana organizations serve similar purposes in the Grafana platform. While both are designed to help manage and control access to resources, teams provide a more focused approach for smaller groups or projects. Teams also enable collaboration and shared ownership of resources among members, making it simple to manage access and control who has access to which resources.
+
+In contrast, organizations provide a higher level of management for multiple teams and resources. They are designed for larger enterprises or organizations with a complex setup or multiple or different business models.
+
+## Access to external systems
+
+Consider the need for machine-to-machine [M2M] communications. If a system needs to interact with Grafana, ensure it has proper access.
+
+Consider the following scenarios:
```

Comment:
thought: these scenarios deal with getting data into Grafana, which is a different use case than what service accounts are for. To get data, you would use data sources. What do you think about changing the examples to different use cases, such as instance provisioning (through Terraform or scripts), alert scheduling etc.

Reply:
oh wait, it looks like @linoman describes service account use cases following this section. Do we need the "Access to external systems" content at all?